internal/testutility/normalize.go

```diff
@@ -110,6 +110,7 @@ func normalizeErrors(t *testing.T, str string) string {
 	str = strings.ReplaceAll(str, "The system cannot find the file specified.", "no such file or directory")
 	str = strings.ReplaceAll(str, "CreateFile", "lstat")
 	str = strings.ReplaceAll(str, "\nstat ./fixtures/", "\nlstat ./fixtures/")
+	str = strings.ReplaceAll(str, "stat <rootdir>", "GetFileAttributesEx <rootdir>")
```

Comment:
We don't need the `<rootdir>`. Also this is the wrong way around. See line 111 for similar example.